### ISSUE TITLE
add ability to search friends

### ISSUE THREAD

**Issue by troke12** [ppy/osu-web#6680]
in [friends page](https://osu.ppy.sh/home/friends) theres no option to search friends, because some of users like me have a lots of friends, so i can't use scroll everytime when i'm searching my friends. Also this is good for in-game too

the design looks like this :

![ss](https://cdn.discordapp.com/attachments/408950289962369025/757777887951847515/unknown.png)

**Comment by peppy** [comment]
as a temporary method, you can hit ctrl-f in your browser to search the page.

**Comment by jacksonisiah** [comment]
Not really sure the usefulness for this as you can just search like normal or go into a smaller view to see them.

**Comment by troke12** [comment]
yeah i can search normal or using smaller view, but how about in-game? 

![ss](https://cdn.discordapp.com/attachments/408950289962369025/757783620390748201/unknown.png)
